packages/account/src/Components/file-uploader-container/__tests__/file-uploader.spec.tsx

```diff
@@ -48,11 +49,11 @@ describe('<FileUploader />', () => {
 
         const file = new File(['hello'], 'hello.png', { type: 'image/png' });
 
-        const input = screen.getByTestId('dt_file_upload_input');
+        const input = screen.getByTestId('dt_file_upload_input') as HTMLInputElement;
         fireEvent.change(input, { target: { files: [file] } });
```

Comment:
maybe better to write in this way:
const input = screen.getByTestId<HTMLInputElement>('dt_file_upload_input');
or
const input: HTMLInputElement = screen.getByTestId('dt_file_upload_input');

Reply:
Will change this